ci/run_travis.sh

```diff
@@ -82,6 +82,7 @@ echo -n "RUNNING UNIT TESTS "
 if running_on_linux; then
   start_processing
   test/unittests/cvmfs_unittests --gtest_shuffle --gtest_filter="-*Slow:T_Dns.CaresResolverLocalhost:T_Dns.NormalResolverCombined:T_Dns.CaresResolverMany" > unittests.log 2>&1  || { UNITTESTS_FAILED=true; report_error "$(pwd)/unittests.log"; }
+  (cd cvmfs/webapi && python test_cvmfs_geo.py) >> unittests.log 2>&1 || { UNITTESTS_FAILED=true; report_error "$(pwd)/unittests.log"; }
```

Comment:
This needs to be cd ../cvmfs/webapi.  I merged this pr into pr #1983, made that change to this line and also added a -v after test_cvmfs_geo.py, and it passed a [travis test](https://travis-ci.org/DrDaveD/cvmfs/jobs/276033753).  My DrDaveD/cvmfs repo is having better luck at getting travis jobs completed right now than cvmfs/cvmfs; there appears to be a lot queued up here.

Reply:
The travis holdup is the osx builds.  Mine is also blocked on that.  There are four later cvmfs/cvmfs builds pending on both platforms right now, waiting for one build to finish its osx.